k8s/base/ingress.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ingress-shiny
+  annotations:
+    nginx.ingress.kubernetes.io/affinity: "cookie"
+    nginx.ingress.kubernetes.io/session-cookie-name: "shinycookie"
+    nginx.ingress.kubernetes.io/session-cookie-expires: "172800"
```

Comment:
Is this in seconds? 48 hours? Seems reasonable

Reply:
yup in seconds... 48 hours